pkg/bundle/multi_scoped_cache/cache.go

```diff
@@ -14,12 +14,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package internal
+package multi_scoped_cache
```

Comment:
We want to avoid underscores in package names as much as we can. Changing the name to `cache` would be better.

https://go.dev/blog/package-names